molecule/mysql_hardening/INSTALL.rst

```diff
@@ -0,0 +1,22 @@
+*******
```

Comment:
this `INSTALL.rst` seems useless, maybe delete it? (also all others in the adjacent molecule directories)